_posts/0000-01-02-AshwinKumar2001.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
![image](https://user-images.githubusercontent.com/72250330/141676410-8d4408f4-c852-4d8d-ba5a-f8d952ed39ba.png)
This should come in handy!